File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int current_sum=0;
        int max_sum=0;
        for(int i:nums){
            if(i==1){ 
                current_sum++;
                max_sum=max(max_sum,current_sum);
            }else{
                current_sum=0;
            }}return max_sum;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        current_sum=0
        max_sum=0
        for i in nums:
            if i==1: 
                current_sum+=1
                max_sum=max(max_sum,current_sum)
            else:
                current_sum=0
        return max_sum



        